docs/conf.py

```diff
@@ -357,4 +357,11 @@
     ("py:class", "aiohttp.SimpleCookie"),  # undocumented
     ("py:class", "aiohttp.web.RequestHandler"),  # undocumented
     ("py:class", "aiohttp.NamedPipeConnector"),  # undocumented
+    ("py:class", "asyncio.AbstractServer"),  # undocumented
+    ("py:mod", "aiohttp.test_tools"),  # undocumented
+    ("py:class", "list of pairs"),  # undocumented
+    ("py:class", "aiohttp.protocol.HttpVersion"),  # undocumented
+    ("py:attr", "self.server.app"),  # undocumented
```

Comment:
This should have an actual importable path. Or maybe there should be no reference at all because this seems to be pointing at an attribute of a runtime instance of something. Probably just turn it into an inline code thingy instead of using an attr role.

Reply:
I add some fixes and resolve outdated conflicts. Converted to draft for a while

Reply:
```suggestion
```